src/card-form.js

```diff
@@ -5,8 +5,8 @@ export default class CardForm extends React.Component {
     super(props)
     this.state = {
       topic: props.selected ? props.selected.topic : '',
-      sideA: props.selected ? props.selected.sideA : '',
-      sideB: props.selected ? props.selected.sideB : '',
+      question: props.selected ? props.selected.question : '',
+      answer: props.selected ? props.selected.answer : '',
```

Comment:
It is possible to simplify these conditional assignments. Try using a default object or default properties in a call to `Object.assign()`.